docker/sbtc/signer/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM sbtc-build:latest AS build
+FROM debian:bookworm-slim AS runtime
+COPY --from=build /code/sbtc/target/debug/sbtc-signer /usr/local/bin/sbtc-signer
+ENTRYPOINT ["/usr/local/bin/sbtc-signer --config /signer-config.toml --migrate-db"]
```

Comment:
All signer containers (after fixing the possible typo above) immediately exit with the following:

```bash
sbtc-signer-1  | Error: missing field `bootstrap_signing_set`
```

https://github.com/stacks-network/sbtc/blob/cfd696783714668bb0ad0ce96925a49ca722c341/docker/sbtc/signer/signer-config.toml#L79 does not seem to include that key.